apps/vx-mark/frontend/src/app_refresh.test.tsx

```diff
@@ -74,12 +80,15 @@ it('Refresh window and expect to be on same contest', async () => {
   });
 
   unmount();
+  apiMock.mockApiClient.assertComplete();
+  apiMock.mockApiClient.reset();
```

Comment:
Why reset here? Is it to help make debugging easier by separating phases of the test?

Reply:
I thought I was getting errors without this but now I'm not so... will remove